client/landing/stepper/declarative-flow/internals/steps-repository/readymade-template-generate-content/components/readymade-template-preview/index.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+	BlockRendererProvider,
+	ReadymadeTemplateRenderer,
+	RenderedContent,
+} from '@automattic/block-renderer';
+import { Spinner } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+import React from 'react';
+import { RENDERER_SITE_ID } from 'calypso/my-sites/patterns/constants';
+import './style.scss';
+import SitePreview from '../../../../components/site-preview';
+
+type RTPreviewProps = {
+	renderedContent?: RenderedContent;
+	isLoading?: boolean;
+	siteSlug?: string | null;
+};
+
+const ReadymadeTemplatePreview: React.FC< RTPreviewProps > = ( {
+	renderedContent,
+	isLoading = false,
+	siteSlug = null,
+} ) => {
+	if ( isLoading ) {
+		const loadingMessage = translate( '{{strong}}Generating content for your site.{{/strong}}', {
```

Comment:
I think we can leave the `<strong>` markup outside of the translation:

const loadingMessage = ( <strong>{ translate( 'Generating content for your site.' ) }</strong> )

Reply:
Sure.